modules/tinymce/src/themes/silver/test/ts/browser/editor/toolbar/ToolbarDrawerToggleTest.ts

```diff
@@ -107,4 +107,27 @@ describe('browser.tinymce.themes.silver.editor.toolbar.ToolbarDrawerToggleTest',
       });
     });
   });
+
+  context(`Should preserve focus if skipFocus: true option was passed`, () => {
```

Comment:
I think you'll need to add tests for when `skipFocus` is explicitly set to false, also.

Reply:
Added the additional test [here](https://github.com/tinymce/tinymce/pull/8272/commits/cc1abae83b76cf99918426cb99b0a854a59208df).